src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx

```diff
@@ -22,22 +25,7 @@ const AdminToolsUserMangaement: React.FC = () => {
   const updateUserRole = useUpdateUserRole();
   const theme = useTheme();
 
-  const autocompleteStyle = {
-    height: '40px',
-    backgroundColor: theme.palette.background.default,
-    width: '100%',
-    borderRadius: '25px',
-    border: 0,
-    '.MuiOutlinedInput-notchedOutline': {
-      borderColor: 'black',
-      borderRadius: '25px'
-    },
-    '&.Mui-focused .MuiOutlinedInput-notchedOutline': {
-      borderColor: 'red'
-    }
-  };
-
-  const selectStyle = {
+  const roleSelectStyle = {
```

Comment:
can you do this like this, I think it makes it easier to add in styles later and it'll be easier to figure out for new ppl if we do it standard like this everywhere. This is my b, for having peyton do it with individual style objects:
```
  const styles = {
    roleSelect: {
      width: '100%',
      backgroundColor: theme.palette.primary.main,
      borderRadius: '25px',
      height: '40px',
      '.MuiOutlinedInput-notchedOutline': { borderRadius: '25px', borderColor: 'black' },
      '&.Mui-focused .MuiOutlinedInput-notchedOutline': { border: 0 },
      '&.Mui-disabled': { backgroundColor: theme.palette.background.paper }
    }
  };